web/src/app/lib/services/burst.service.ts

```diff
@@ -258,4 +260,148 @@ export class BurstService {
             return nameKey.replace(/_/g, " ");
         }
     };
+
+
+    public getTransactionNameFromType(transaction: Transaction, account: Account) {
```

Comment:
Wow....this screams for refactoring...time will come.

Reply:
Yeah... 🙄